playbooks/install_software.yml

```diff
@@ -2,20 +2,8 @@
 - name: Install software
   hosts: all
   vars_files:
-    - /home/user/power-up/playbooks/poweraie_software.yml
```

Comment:
The 'packages' var used on line 23 comes from this 'poweraie_software.yml' so it shouldn't be removed. That said my install playbook patch re-works this so it won't really matter.